doc-content/jbpm-docs/src/main/asciidoc/Integration/integration-maven-dependency-con.adoc

```diff
@@ -0,0 +1,177 @@
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <version.org.drools>{MAVEN_ARTIFACT_VERSION}</version.org.drools>
+    <version.org.jbpm>{MAVEN_ARTIFACT_VERSION}</version.org.jbpm>
+    <hibernate.version>5.3.17.Final</hibernate.version>
+    <hibernate.core.version>5.3.17.Final</hibernate.core.version>
+    <slf4j.version>1.7.26</slf4j.version>
+    <jboss.javaee.version>1.0.0.Final</jboss.javaee.version>
+    <logback.version>1.2.9</logback.version>
+    <h2.version>1.3.173</h2.version>
+    <narayana.version>5.9.0.Final</narayana.version>
+    <jta.version>1.0.1.Final</jta.version>
+    <junit.version>4.13.1</junit.version>
+  </properties>
+  <dependencyManagement>
+    <dependencies>
+      <!-- define Drools BOM -->
+      <dependency>
+        <groupId>org.drools</groupId>
+        <artifactId>drools-bom</artifactId>
+        <type>pom</type>
+        <version>${version.org.drools}</version>
+        <scope>import</scope>
+      </dependency>
+      <!-- define jBPM BOM -->
+      <dependency>
+        <groupId>org.jbpm</groupId>
+        <artifactId>jbpm-bom</artifactId>
+        <type>pom</type>
+        <version>${version.org.jbpm}</version>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+----
+
+In modules that use the {PROCESS_ENGINE} Java API (KIE API), declare the necessary {PROCESS_ENGINE} dependencies and other components that the modules require, as in the following example:
+
+.Dependencies for modules that use the KIE API
+[source,xml]
+----
+    <dependency>
+      <groupId>org.jbpm</groupId>
+      <artifactId>jbpm-flow</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.jbpm</groupId>
+      <artifactId>jbpm-flow-builder</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.jbpm</groupId>
+      <artifactId>jbpm-bpmn2</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.jbpm</groupId>
+      <artifactId>jbpm-persistence-jpa</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.jbpm</groupId>
+      <artifactId>jbpm-human-task-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.jbpm</groupId>
+      <artifactId>jbpm-runtime-manager</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>${slf4j.version}</version>
+    </dependency>
+----
+
+If your application uses persistence and transactions, you must add artifacts that implement the JTA and JPA frameworks. Additional dependencies are required for testing the workflow components before actual deployment.
+
+The following example defines the dependencies that include Hibernate for JPA, the H2 database for persistence, Narayana for JTA, and the components needed for testing. This example uses the `test` scope. Adjust this example as necessary for your application; for production use, remove the `test` scope.
```

Comment:
Suggest "... your application. For product use, ..." I don't think the semicolon is called for and may even be incorrect in this case.